tensorflow_model_optimization/python/core/clustering/keras/BUILD

```diff
@@ -0,0 +1,111 @@
+package(default_visibility = [
+    "//tensorflow_model_optimization:__subpackages__",
+])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+py_library(
+    name = "keras",
+    srcs = [
+        "__init__.py",
+    ],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":cluster",
+    ],
+)
+
+py_library(
```

Comment:
Note for self (you can ignore for now): internally we have checks to require that all deps are included directly (e.g. in this [fashion](https://github.com/tensorflow/model-optimization/blob/f13568b7bdfa8afd5e945fbc150de0a0764e14ab/tensorflow_model_optimization/python/core/sparsity/keras/BUILD), now simplified after [this commit](https://github.com/tensorflow/model-optimization/commit/ff464f9771e593393479941d76cb45b81c7d2fe3)).

Going to remind myself what the benefits are and then see if there's an easier way to contribute such changes if still needed.